### PR TITLE
core::util-linux: bump to v2.40.2

### PR DESCRIPTION
**Pull request by sixtyfourktec**
Add a new generic bin target and install only libs in *-tgt. Some apps depend on the libs and don't necessarily need the executables as well. Also there was a provideDeps missing.

### CODE REVIEW

File: recipes/core/util-linux.yaml [hunk 1]
```diff
@@ -34,11 +34,19 @@ buildScript: |
...
     dev:
         packageScript: autotoolsPackageDev
+        provideDeps: [ "*-dev" ]
 
     tgt:
-        packageScript: autotoolsPackageTgt
+        packageScript: autotoolsPackageLib
         provideDeps: [ "*-tgt" ]
 
     tools:
```
Comment (sixtyfourktec) [inline recipes/core/util-linux.yaml:52]: Actually we also could remove the extra tools target and move this into the generic bin target. What do you think?

Comment (jkloetzke) [reply recipes/core/util-linux.yaml:52]: Yes, this sounds reasonable. :+1:

File: recipes/core/util-linux.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ inherit: [autotools]
 
 metaEnvironment:
     PKG_VERSION_MAJOR: "2.40"
-    PKG_VERSION: "2.40.1"
+    PKG_VERSION: "2.40.2"
```
Comment (jkloetzke) [inline recipes/core/util-linux.yaml:5]: The version bump should go into a separate commit.

File: recipes/core/util-linux.yaml (hunk 1 above)
Comment (sixtyfourktec) [reply recipes/core/util-linux.yaml]: done

**Comment by jkloetzke** [comment]
Thanks!
